src/index.html

```diff
@@ -1,13 +1,424 @@
+                  <a href="#" class="link">
+                    Max Henry
+                  </a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Adress
+                </div>
+                <a
+                  href="https://goo.gl/maps/UQb41HBW5QkM6JN98"
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="GET">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  required
+                  class="contact-us__field form-field"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  required
+                  class="contact-us__field form-field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  required
+                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
+                >
+                </textarea>
```

Comment:
```suggestion
               ></textarea>
```

Існує баг з textarea, коли не відображається плейсхолдер. А якщо вводити текст, то в нього якесь дивне позиціонування. Це фікситься використанням такого форматування тегу, як я вказав (в один рядок і 1 відступ вліво)

<img width="549" alt="Снимок экрана 2021-10-07 в 16 01 43" src="https://user-images.githubusercontent.com/66570142/136389454-d24f5b2b-3411-4e5d-bd8f-128eae2e23b6.png">
<img width="549" alt="Снимок экрана 2021-10-07 в 16 01 46" src="https://user-images.githubusercontent.com/66570142/136389458-435f2112-7462-4cf3-91e5-9b85c433a13a.png">